pkg/enforcement/user_permissions.go

```diff
@@ -11,7 +11,9 @@ import (
 
 type TenantUserPermissions struct {
 	Tenant      TenantDetails `json:"tenant"`
+	Resource    *Resource     `json:"resource,omitempty"`
```

Comment:
This is not the correct type. The right type is `ResourceDetails` that contains only `key` `type` and `attributes`
https://pdp-api.permit.io/redoc#tag/Authorization-API/operation/Get_User_Permissions_user_permissions_post

 ```suggestion
	Resource    *ResourceDetails     `json:"resource,omitempty"`
```